action.yml

```diff
@@ -18,7 +18,7 @@ inputs:
     required: false
     default: "2"
   exit:
-    description: "Wether the coverage check should exit."
+    description: "Weather the coverage check should exit."
```

Comment:
Het is "[whether or not](https://www.merriam-webster.com/dictionary/whether%20or%20not)". Dus, hier moet het zijn: "Whether"

![image](https://user-images.githubusercontent.com/59729886/200591692-072c1f14-a89d-45c4-857a-639d80b12877.png)

Reply:
aangepast